docs/getting_started/getting_started.md

```diff
@@ -10,6 +10,6 @@ To get started working with [[= product_name =]], see how you can get an install
 [[= cards([
     "getting_started/requirements",
     "getting_started/install_ibexa_dxp",
-    "getting_started/install_on_ibexa_cloud",
+    "ibexa_cloud/install_on_ibexa_cloud",
```

Comment:
I'm wondering if we need it here.
Since Ibexa Cloud will have its own hub page, it will be easy to find in ToC.
Installing Ibexa Cloud is not necessary to get started with Ibexa DXP. We don't have a tile here with, for example, CDP installation, as it's also something additional.
```suggestion

```

Reply:
Quoting myself, "Installing on Ibexa Cloud is a hard way to get started."